engine/src/main/java/org/terasology/rendering/dag/nodes/AlphaRejectBlocksNode.java

```diff
@@ -151,9 +167,9 @@ public void process() {
         chunkMaterial.setInt("textureAtlas", 0, true);
         chunkMaterial.setInt("textureEffects", 1, true);
         chunkMaterial.setInt("textureLava", 2, true);
-        if (renderingConfig.isNormalMapping()) {
+        if (isNormalMapping) {
             chunkMaterial.setInt("textureAtlasNormal", 3, true);
-            if (renderingConfig.isParallaxMapping()) {
+            if (isParallaxMapping) {
```

Comment:
Yeah, I'm aware that Parallax Mapping doesn't work, and also of some of the glitches associated with it. I hope to dive into it as soon as I get past the preliminary work :D
( Hopefully I won't have to wait for ~4 years :P )